libs/payments/stripe/src/lib/stripe.util.spec.ts

```diff
@@ -36,11 +35,9 @@ describe('util', () => {
       const mockPrice = StripePriceFactory();
       const mockPromoCode = 'promo_code1';
 
-      try {
-        checkSubscriptionPromotionCodes(mockPromoCode, mockPrice, undefined);
-      } catch (error) {
-        expect(error).toBeInstanceOf(PromotionCodeNotForSubscriptionError);
-      }
+      expect(() =>
+        checkSubscriptionPromotionCodes(mockPromoCode, mockPrice, undefined)
+      ).toThrowError(PromotionCodeNotForSubscriptionError);
```

Comment:
How come these aren't doing the same `.rejects` style that was done above with an `await` in front?

Reply:
Because the other is the jest format for handling async code, and this is the jest format for handling synchronous code.